medusa/search/manual.py

```diff
@@ -34,14 +36,19 @@
 SEARCH_STATUS_SEARCHING = 'searching'
 
 
+# Get key names from `common.Quality` class (for the quality values and any-sets)
+# Copied from `medusa/server/api/v2/config.py` @ `DataGenerator.data_consts`
+quality_key_map = {
```

Comment:
Same as below.

Reply:
If I move this to Quality class then there's no need for it.
I mean... I still need this map, but I probably would not generate it from `Quality.__dict__`, I will just make a static dict for it...
Yeah maybe I'll do that.